its/pom.xml

```diff
@@ -81,6 +81,7 @@
             <includes>
               <include>com/sonar/it/csharp/Tests.java</include>
               <include>com/sonar/it/vbnet/Tests.java</include>
+              <include>com/sonar/it/shared/ScannerCliTest.java</include>
```

Comment:
I manually ran this test and it was failing because there were 2 new warnings in the logs (and this test is checking `containsExactlyInAnyOrder` for the WARN logs):

```
12:27:28.557 WARN: Access to the multi-values/property set property 'sonar.cs.analyzer.projectOutPaths' should be made using 'getStringArray' method. The SonarQube plugin using this property should be updated.
```